src/app/core/store/configuration/configuration.effects.ts

```diff
@@ -49,11 +50,14 @@ export class ConfigurationEffects {
       this.stateProperties.getStateOrEnvOrDefault<string>('ICM_APPLICATION', 'icmApplication'),
       this.stateProperties
         .getStateOrEnvOrDefault<string | string[]>('FEATURES', 'features')
-        .pipe(map(x => (typeof x === 'string' ? x.split(/,/g) : x)))
+        .pipe(map(x => (typeof x === 'string' ? x.split(/,/g) : x))),
+      this.stateProperties
+        .getStateOrEnvOrDefault<string | Theme[]>('THEME', 'themes')
+        .pipe(map(x => (typeof x === 'string' ? x : x.find(t => t.default).name)))
```

Comment:
please also rename theme `orange` to `default` and fallback to it here, if the incoming property is falsy.